<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherControlsDialog</class>
 <widget class="QDialog" name="OtherControlsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>293</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#OtherContrilsDialog{
	background: transparent;
}</string>
  </property>
  <widget class="QWidget" name="cWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#cWidget{
	background: rgb(65, 65, 65);
    /*border-radius: 8px;*/
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="titleWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: black;
border: none;
font-family: 'Microsoft YaHei';
font-size: 12px;
color: white;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>其它操作</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>435</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_close">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">btn_close {
    border: none;
    background: transparent;
}

btn_close::hover {
    background: #345343;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../Img.qrc">
           <normaloff>:/Resources/img/window_close.png</normaloff>:/Resources/img/window_close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: white;
    font-family: 'Microsoft YaHei';
	font: 12px;
}
QPushButton {
    font-family: 'Microsoft YaHei';
    background: #303133;
    border-radius: 4px;
	font: 12px;
    color:white;
}
QPushButton::hover {
    font-family: 'Microsoft YaHei';
    background: #606060;
    border-radius: 4px;
	font: 12px;
    color:white;
}
QPushButton::pressed {
    font-family: 'Microsoft YaHei';
    background: #82848A;
    border-radius: 4px;
	font: 12px;
    color:white;
}
</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QPushButton" name="btn_enable_3">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_stop_2">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_enable_1">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btn_resetPos_5">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btn_stop_5">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>探测器Z轴</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btn_stop_6">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>探测器X轴</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_stop_1">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="btn_stop_7">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="btn_enable_7">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_resetPos_3">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>光管Z轴</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_resetPos_1">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btn_stop_4">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="btn_resetPos_7">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>探测器Y轴</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>旋转轴</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>载物台Y轴</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn_enable_4">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>载物台X轴</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_resetPos_2">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btn_resetPos_6">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_enable_2">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_resetPos_4">
         <property name="text">
          <string>回零</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_stop_3">
         <property name="text">
          <string>轴停止</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btn_enable_5">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btn_enable_6">
         <property name="text">
          <string>断开使能</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Img.qrc"/>
 </resources>
 <connections/>
</ui>
